lib/default_assign_issue_patch.rb

```diff
@@ -6,16 +6,21 @@ def self.included(base) # :nodoc:
 
     # Same as typing in the class
     base.class_eval do
-      unloadable
+      unloadable # Send unloadable so it will not be unloaded in development
 
-      before_create :assign_default_assignee
+      before_save :assign_default_assignee
```

Comment:
I'm worried that this will affect issues that have already been created. For instance, if someone creates an issue an deliberately sets a blank assignee, the plugin will attempt to set an assignee every time the issue is updated. I think the more expected behaviour is for the plugin to affect only new issues. What do you think?